### ISSUE TITLE
❌ Error in estimate_depth_intrinsic: module 'utils3d.torch' has no attribute 'depth_edge'

### ISSUE THREAD

**Issue by Rorschach2333** [GZWSAMA/One23Pose#28]
❌ Error in estimate_depth_intrinsic: module 'utils3d.torch' has no attribute 'depth_edge'

 refer to this post:
https://github.com/henry123-boy/SpaTrackerV2/issues/29
 change 'depth_edge' to 'depth_map_edge' in SpaTrack.py
and do this in line 537:
unc_metric *= (~(utils3d.torch.depth_map_edge(metric_depth_gt.squeeze(), rtol=0.03, mask=mask_roi.squeeze().bool()))).float().reshape(*metric_depth_gt.shape)

**Comment by Rorschach2333** [comment]
also pip install pyceres==2.4, if you meet 
"❌ Error in estimate_depth_intrinsic: 'pycolmap._core.Image' object has no attribute 'id'"

**Comment by Rorschach2333** [comment]
pin to this util3d version
git+https://github.com/EasternJournalist/utils3d.git@d3a577acf0a9ad7e513a1416449a07b6f47d967f#egg=utils3d

**Comment by Rorschach2333** [comment]
for this warning:
rch/utils/cpp_extension.py:1965: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
In app.py:

import torch
os.environ['TORCH_CUDA_ARCH_LIST'] = '.'.join(map(str,torch.cuda.get_device_capability())) #=''
#os.environ['TORCH_CUDA_ARCH_LIST']='9.0'
